apps/managedidentity/managedidentity_test.go

```diff
@@ -661,14 +661,20 @@ func TestAppServiceAcquireTokenReturnsTokenSuccess(t *testing.T) {
 }
 
 func TestAzureMLAcquireTokenReturnsTokenSuccess(t *testing.T) {
+	// Set the DEFAULT_IDENTITY_CLIENT_ID environment variable
```

Comment:
please forgive my nitpicking, but this code doesn't warrant a comment. You can assume readers understand `Setenv()`